src/ZeroLog.sln

```diff
@@ -29,6 +29,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ZeroLog.Analyzers", "ZeroLo
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ZeroLog.Analyzers.Tests", "ZeroLog.Analyzers.Tests\ZeroLog.Analyzers.Tests.csproj", "{FC5C8744-6E2C-49B0-9B3B-6BE6F825DF6C}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ZeroLog.Tests.ExternalAppender", "ZeroLog.Tests.ExternalAppender\ZeroLog.Tests.ExternalAppender.csproj", "{92721CF3-E8E7-420B-9779-DCDDA8A336D9}"
```

Comment:
Oops, I wanted to delete this project but I apparently only removed it from the solution. It was used to make sure an external appender could be instantiated from the JSON config (that I removed).

Do you plan to use it, or should it be deleted?

Reply:
The solution wasn't building without it...

Reply:
Oh? It compiled fine for me... I'll make sure to delete it properly then.